From 11bf73087008d636ce7884a4fd74829bfc11a3f4 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Wed, 28 Dec 2022 01:01:38 +0800
Subject: [PATCH 16/28] Revert "drm/amd/pm: disable BACO entry/exit completely
 on several sienna cichlid cards"

This reverts commit f3adf0adf306e82c343206f5cfa87cccf3ca6f82.
---
 .../gpu/drm/amd/pm/swsmu/smu11/sienna_cichlid_ppt.c   | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/drivers/gpu/drm/amd/pm/swsmu/smu11/sienna_cichlid_ppt.c b/drivers/gpu/drm/amd/pm/swsmu/smu11/sienna_cichlid_ppt.c
index 91026d0c1c798..def32b6897f99 100644
--- a/drivers/gpu/drm/amd/pm/swsmu/smu11/sienna_cichlid_ppt.c
+++ b/drivers/gpu/drm/amd/pm/swsmu/smu11/sienna_cichlid_ppt.c
@@ -309,17 +309,6 @@ static void sienna_cichlid_check_bxco_support(struct smu_context *smu)
 		smu_baco->platform_support =
 			(val & RCC_BIF_STRAP0__STRAP_PX_CAPABLE_MASK) ? true :
 									false;
-
-		/*
-		 * Disable BACO entry/exit completely on below SKUs to
-		 * avoid hardware intermittent failures.
-		 */
-		if (((adev->pdev->device == 0x73A1) &&
-		    (adev->pdev->revision == 0x00)) ||
-		    ((adev->pdev->device == 0x73BF) &&
-		    (adev->pdev->revision == 0xCF)))
-			smu_baco->platform_support = false;
-
 	}
 }
 
-- 
2.25.1

